### ISSUE TITLE
Implementation of userflow for staking

### ISSUE THREAD

**Issue by joyboyisfree** [thevantageproject/PolkaAnalytics#58]
Dependent on #56

**Comment by joyboyisfree** [comment]
Closing in favor of https://github.com/buidl-labs/YieldScan/issues?q=is%3Aopen+is%3Aissue+label%3Aui-ux
